docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -103,11 +92,12 @@ You may want to reach the maximum developers and projects, so use .NET Framework
 .NET Framework 4.5.2 --> net452
 .NET Framework 4.6   --> net46
 .NET Framework 4.6.1 --> net461
+.NET Framework 4.6.2 --> net462
 ```
 
-For example, here's how you would write a library which targets .NET Framework 4.0:
+For example, here's how you would write a library which targets .NET Framework 4:
```

Comment:
it would be good to be consistent with the version throughout the topic. .NET Framework 4 is the most common brand name instead of 4.0 (see https://msdn.microsoft.com/en-us/library/ff361664(v=vs.110).aspx). Here you fixed it but there other places still with 4.0.

Also
.NET Framework 4 -> the .NET Framework 4